### PR TITLE
Update installation_guide.rst

### PR DESCRIPTION
**Pull request by Cebrain**
How to fix wrong time in Profit / Loss report from Docker.

### CODE REVIEW

File: docs/installation_guide.rst [hunk 1]
```diff
@@ -151,6 +151,20 @@ At this point the rotki docker container should be running and you
 should be able to access rotki frontend, open your browser and go to
 it at ``http://localhost:8084``. You should be able to see the rotki login screen.
 
+Time in Profit / Loss Report is in wrong.
```
Comment (yabirgb) [inline docs/installation_guide.rst:154]: This change doesn't affect only to the Profit and Loss Report but to every action. I believe a better title is

```suggestion
Modify the timezone used by rotki
```

File: docs/installation_guide.rst [hunk 2]
```diff
@@ -151,6 +151,20 @@ At this point the rotki docker container should be running and you
...
+---------------------------
+To fix this problem, it is necessary to set the TimeZone enviroment variable with the command ``-e TZ=Timezone`` when starting the container::
```
Comment (yabirgb) [inline docs/installation_guide.rst:156]:
```suggestion
To set the timezone in the docker environment you can use the option ``-e TZ=Timezone`` when starting the container:
```

File: docs/installation_guide.rst (hunk 1 above)
Comment (Cebrain) [reply docs/installation_guide.rst:154]: The UI is running in the correct time zone also with no timezone set in the container.
But your title should also be fine.

**Review by yabirgb — APPROVED** [review]
Thanks

**Pull request merged**